app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if(params[:uuid] == nil)
+
+      channels = Channel.all.map { |channel| channel.id }
+      data = JSON.pretty_generate(channels)
+      render(json: data)
+
+    else
+
+      channel = Channel.find_by_id(params[:uuid])
+
+      if(channel == nil)
+
+        error_response = JSON.pretty_generate({
+          errors: [{
+            status: "404",
+            title: "Not Found",
+            detail: "Channel with uuid " + params[:uuid] + " not found"
```

Comment:
String interpolation would probably make more sense here
`"Channel with uuid #{params[:uuid]} not found"`

Reply:
Agree, it's cleaner, fixed.